.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+## How to contribute
+
+Share your knowledge with the community by contributing to Magento DevDocs!
+You can contribute by creating an issue or pull request (PR) on our [devdocs](https://GitHub.com/magento/devdocs){:target="_blank"} GitHub repository.
+We welcome all types of contributions; from minor typo fixes to new topics.
+
+Magento's tech writer team and [Maintainers]({{ page.baseurl }}/contributor-guide/devdocs-maintainers.html) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
+Working through the backlog takes time, though, so we appreciate your patience.
+
+## Rewards and thanks
```

Comment:
Suggest adding this link somewhere in this section:
https://github.com/magento/magento2/wiki/Contribution-Rewards

Reply:
Adding to this section